packages/carbon/src/classes/Client.ts

```diff
@@ -160,6 +183,21 @@ export class Client {
 		if (options.autoDeploy) {
 			this.handleDeployRequest()
 		}
+
+		this.log(
+			`Carbon initalized with routes:${this.routes
+				.filter((x) => !x.disabled)
+				.map((x) => {
+					return `\n\t${x.method} ${x.path}`
+				})}`
+		)
```

Comment:
In serverless, the client is constructed for every request, meaning this log is called on every request and would spam the logs